cucumber-expressions/javascript/test/cucumber_expression_test.js

```diff
@@ -63,7 +63,7 @@ describe(CucumberExpression.name, () => {
   })
 
   describe('RegExp special characters', () => {
-    ['[', '\\', '^', '$', '?', '*', '+'].forEach((character) => {
+    ['\\', '[', ']', '^', '$', '.', '|', '?', '*', '+'].forEach((character) => {
```

Comment:
I'd rather comment why those tests were excluded then have them in if they aren't needed.